spec/factories/patients.rb

```diff
@@ -91,7 +91,16 @@
     family_name { Faker::Name.last_name }
     date_of_birth do
       if year_group
-        Faker::Date.birthday(min_age: year_group + 5, max_age: year_group + 5)
+        # Calculate academic year start (September 1st)
+        academic_year_start = Date.new(Date.current.year, 9, 1)
+        # Adjust year to previous September if we're before September
+        academic_year_start = academic_year_start.prev_year if Date.current <
+          academic_year_start
```

Comment:
```suggestion
        academic_year_start = Date.new(Date.current.academic_year, 9, 1)
```

I think this is equivalent.

Reply:
Ah, nice, thanks – incorporated.